common/src/types/v0/transport/node.rs

```diff
@@ -105,6 +105,21 @@ impl Default for NodeStatus {
     }
 }
 
+#[derive(Serialize, Deserialize, Debug, Clone, EnumString, ToString, Eq, PartialEq)]
+pub enum DrainStatus {
+    /// Node has no drain cordon labels
+    NotDraining,
+    /// Node is draining
+    Draining,
+    /// Node is drained
+    Drained,
+}
+impl Default for DrainStatus {
+    fn default() -> Self {
+        Self::NotDraining
```

Comment:
I think with the new rust version you can simply add #[default] above the NodeDraining field in the enum itself - I've not tested this myself

Reply:
seems to work :+1: